providers/openshift/provider.go

```diff
@@ -140,33 +140,34 @@ func (p *OpenShiftProvider) setCA(paths []string) error {
 	return nil
 }
 
+// encodeSARWithScope adds a "scopes" array to the SAR if it does not have one already, and outputs
+// as pretty encoding.
+func encodeSARWithScope(json *simplejson.Json) ([]byte, error) {
+	if len(json.Get("scopes").MustArray()) == 0 {
+		json.Set("scopes", []interface{}{})
+	}
+	return json.EncodePretty()
+}
+
 // parseSubjectAccessReviews parses a list of SAR records and ensures they are properly scoped.
 func parseSubjectAccessReviews(review string) ([]string, error) {
 	if len(review) == 0 {
 		return nil, nil
 	}
+
+	// Convert to a json array to simplify encoding later
+	if review[0] != '[' && review[len(review)-1] != ']' {
+		review = "[" + review + "]"
```

Comment:
Not sure if you want to prestrip any extraneous white space at the start of this function?